src/fn/to92.ts

```diff
@@ -21,39 +29,60 @@ const generate_semicircle = (
   })
 }
 
-export const to92_def = z.object({
-  fn: z.string(),
-  p: length.optional().default("1.27mm"),
-  id: length.optional().default("0.72mm"),
-  od: length.optional().default(".95mm"),
-  w: length.optional().default("4.5mm"),
-  h: length.optional().default("4.5mm"),
-  inline: z.boolean().optional().default(false),
-})
-
-export type To92Def = z.input<typeof to92_def>
-
-export const to92 = (
-  raw_params: To92Def,
-): { circuitJson: AnySoupElement[]; parameters: any } => {
-  const parameters = to92_def.parse(raw_params)
+export const to92_3 = (parameters: z.infer<typeof to92_def>) => {
```

Comment:
What's the point of having a suffix `-3`? There is no footprint with `TO-92-3`. It's just that the default `TO-92` is the one having 3 plated_holes